metadata.rb

```diff
@@ -1,23 +1,15 @@
-# frozen_string_literal: false
-
-require 'semver'
-
-name             'atlantis'
-maintainer       'Ben Abrams'
-maintainer_email 'me@benabrams.it'
+name 'atlantis'
+source_url       'https://github.com/sous-chefs/atlantis'
+issues_url       'https://github.com/sous-chefs/atlantis/issues'
+maintainer       'Sous Chefs'
+maintainer_email 'help@sous-chefs.org'
+chef_version     '>= 15'
```

Comment:
Do we need to break backwards compatibility on intake? I know you want to bring it up to speed with the "standards" but because of the complications around chef 15 and open source users I want to be sensitive to not breaking something they need. My own org only migrated in the last month.

Reply:
oh yeah i can drop this back down. i just copy pasted a bunch of stuff. not sure how dual licensing and chef cookbooks work for now ill leave it as MIT and let the board figure out the licensing deal. i dont think i broke anything else with backwards compatibility so we should be good.